cmake/Modules/FindTileDB_EP.cmake

```diff
@@ -64,8 +64,8 @@ if (NOT TILEDB_FOUND)
           endif()
 
         else() # Linux
-          SET(DOWNLOAD_URL "https://github.com/TileDB-Inc/TileDB/releases/download/2.17.4/tiledb-linux-x86_64-2.17.4-a1f648e.tar.gz")
-          SET(DOWNLOAD_SHA1 "7849cbbf19f5f9252e4be5350328754b01938aaa")
+          SET(DOWNLOAD_URL "https://github.com/TileDB-Inc/TileDB/releases/download/2.17.4/tiledb-linux-x86_64-noavx2-2.17.4-a1f648e.tar.gz")
```

Comment:
I believe you'll need to add an alternate URL for the other set of binaries in order for grade to use them, right?